app/src/main/index.js

```diff
@@ -98,7 +98,11 @@ function handleCrash(error) {
   afterBooted(() => {
     if (mainWindow) {
       mainWindow.webContents.send("error", {
-        message: error ? error.message : undefined
+        message: error
+          ? error.message
+            ? error.message
+            : error
+          : "An unspecified error occurred"
```

Comment:
can we give any more info?

Reply:
++

Reply:
If there is no error, we can not guess the error. ;)
We can change the error message to sth. More meaningful maybe? Do u guys have sth in mind?

Reply:
what are the circumstances where this could happen?

Reply:
i.e. an uncaught `Promise.reject` without error message. We should avoid having those, but still have the fallback if we forget it.